packages/account/src/Sections/Security/TwoFactorAuthentication/two-factor-authentication.jsx

```diff
@@ -55,14 +55,13 @@ const TwoFactorAuthentication = ({ email_address, is_switching, logoutClient })
     const setEnabled = React.useCallback(
         is_enabled => {
             setTwoFactorEnabled(is_enabled);
-            if (!is_enabled) generateQrCode();
```

Comment:
After enabling  or disabling 2FA, the authorizaton token becomes invalid, So this request will result in invalid token. On enabling or disabling 2FA, we are logging out client, so this request is not required.